lib/bouncer.php

```diff
@@ -2,6 +2,27 @@
 
 class Bouncer {
 
+    private static function getChildren($allowed, Page $page) {
```

Comment:
Should this be `Kirby\Cms\Page` instead of `Page`?

Reply:
oh yes, just pushed an additional commit